### ISSUE TITLE
Need refresh when there is a change in the file

### ISSUE THREAD

**Issue by lorenzadam** [bil-elmoussaoui/nautilus-git#28]
Every time you make changes and want to see what changes happen we need to refresh the folder first.

**Comment by bilelmoussaoui** [comment]
Yeah, there's already a refresh whenever you change the branch. And I don't want to refresh Nautilus each x seconds. As Nautilus doesn't provide a way to only refresh the added widget by this extension. Also, git doesn't store any information about modified files until they are stagged, before that i can't know which file was modified unless i add a watcher for each file under git directory which will make your file manager extremely slow (compared to without auto refresh)
